docs/examples/FixedOptions.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+import Select, { components } from '../../src';
+import { colourOptions } from '../data';
+
+type State = {
+  value: [{ [string]: string }]
+};
+
+const TagStyle = {
+  multiValue: (base, state) => {
+    return state.data.isFixed ? { ...base, backgroundColor: 'gray' } : base;
+  },
+  multiValueLabel: (base, state) => {
+    return state.data.isFixed ? { ...base, fontWeight: 'bold', color: 'white', paddingRight: 6 } : base;
+  },
+  multiValueRemove: (base, state) => {
+    return state.data.isFixed ? { ...base, display: 'none' } : base;
+  }
+};
+
+const ClearIndicator = (props) => {
```

Comment:
Rather than providing a conditional component, could you move this logic to `render()` and use the `isClearable` property?